content/source/docs/enterprise/private/vault.html.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Customer Managed Vault (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-vault"
+---
+
+# Private Terraform Enterprise Customer Managed Vault
+
+Private Terraform Enterprise can be configured to use an external Vault cluster
+rather than the internal Vault instance. Within PTFE, Vault is used to
+encrypt sensitive information such as variables and states.
+
+An external Vault cluster allows the customer to have full control over how
+the Vault cluster is managed, for example how it is sealed and unsealed, replicated, etc.
+
+When an external Vault cluster is configured along with the External Services installation mode,
+the product becomes fully stateless and can be run in a hot-standby
+configuration to provide failover.
+
+## Setup
+
+Follwo these steps to configure an external Vault instance:
+
+1. Enable AppRole: `vault auth-enable approle`
+1. Enable transit: `vault mount transit`
+1. Install the `ptfe` policy (See below for policy):
+   `vault policy-write ptfe ptfe.hcl`
+1. Create an AppRole instance:
+   `vault write auth/approle/role/ptfe policies="ptfe"`.
+1. Retrieve the AppRole `role_id`: `vault read auth/approle/role/ptfe/role-id`
+1. Retrieve the AppRole `secret_id`:
+   `vault write -f auth/approle/role/ptfe/secret-id`
+
+## role\_id and secret\_id
+
+The `role_id` and `secret_id` values created during configuration will be input during
+the PTFE installation along with the Vault cluster URL such as
+`https://vault.mycompany.com:8200`. If you use SSL to
```

Comment:
what part of the install process are these asked for?

also will we still be able to automate the install, are these passed in as install data?

Reply:
Yes it can be automated just like the rest of the values.

In the installer UI, it's on the same page as the installation type.